src/shared/libfido2-util.c

```diff
@@ -450,6 +450,19 @@ static int fido2_use_hmac_hash_specific_token(
         return 0;
 }
 
+static const char *fido2_algorithm_to_string(int alg) {
+        switch(alg) {
+                case COSE_ES256:
+                        return "ES256";
+                case COSE_RS256:
+                        return "RS256";
+                case COSE_EDDSA:
+                        return "EDDSA";
```

Comment:
libfido2 appears to also know COSE_ECDH_ES256?

Reply:
No, `fido_cred_set_type` only accept three values as the man page states

> The fido_cred_set_type() function sets the type of cred to cose_alg, where cose_alg is COSE_ES256,
> COSE_RS256, or COSE_EDDSA. The type of a credential may only be set once. Note that not all authentica‐
> tors support COSE_RS256 or COSE_EDDSA.

Reply:
might be good adding a comment to this function saying that this function only maps the stuff usable for creds. And thus COSE_ECDH_ES256 is not listed.